### ISSUE TITLE
Filters.exists() and Filters.notExists() should be swapped 

### ISSUE THREAD

**Issue by SutnuiBorshik** [nikolovskimilos/redistimeseries-js#11]
`Filter.exists('label_1')` assumes the we are looking for samples that belongs to key, containing label with name `label_1`. But right now (version 2.0.0) we receive opposite result, containing samples that belong to key without such label.

Code to reproduce:
```
const timestamp = Date.now();
await rtsClient.add('key1', timestamp, 1).labels({ l1: 1 }).send();
await rtsClient.add('key2', timestamp, 1).labels({ l1: 1, l2: 2 }).send();
const data = await rtsClient.mrange(timestamp, timestamp + 1)
     .filter([
         Filter.equal('l1', 1),
         Filter.exists('l2'),
    ])
    .withLabels()
    .send();
console.log(JSON.stringify(data));
```
Which returns sample that belongs to key without `l2` label:
`[ ["key1", [ ["l1","1"] ] , [ [1590587937150, "1"] ] ] ]`

This is very confusing and acts contrary to docs [https://oss.redislabs.com/redistimeseries/commands/#filtering](https://oss.redislabs.com/redistimeseries/commands/#filtering)

**Comment by gkorland** [comment]
I ran the following and it worked as expected:
```js
const RedisTimeSeries = require('../index'),
        Filter = require('../src/commands/utils/Filter');
const rtsClient = new RedisTimeSeries();

const start = async () => {
  await rtsClient.connect();

  const timestamp = Date.now();
  await rtsClient.add('key1', timestamp, 1).labels({ l1: 1 }).send();
  await rtsClient.add('key2', timestamp, 1).labels({ l1: 1, l2: 2 }).send();
  const data = await rtsClient.mrange(timestamp, timestamp + 1)
     .filter([
         Filter.equal('l1', 1),
         Filter.exists('l2'),
    ])
    .withLabels()
    .send();
  console.log(JSON.stringify(data));
};

start();
```

```bash
$ node example01.js 
[["key1",[["l1","1"]],[[1590606702501,"1"]]],["key2",[["l1","1"],["ls","2"]],[[1590606702501,"1"]]]]
```

Which version of RedisTimeSeries are you using?

**Comment by SutnuiBorshik** [comment]
@gkorland your result is weird as it contains samples from both `key2` with `l2` label and `key1` that doesn't have such label.

I use `redislabs/redistimeseries` docker image. Tried versions 1.2.5 and latest 1.2.6. Both produce same result. But this issue isn't connected to RedisTimeSeries itself but rather with `redistimeseries-js` package.

The problem is that when you use `Filter.exists('label_name')` it is interpreted into
```
label_name=
```
But according to [docs ](https://oss.redislabs.com/redistimeseries/commands/#filtering) it should be
```
label_name!=
```
So filters `exists/notExists` implementations should be swapped. For example package [redis-time-series-ts](https://www.npmjs.com/package/redis-time-series-ts) implements this filters as expected.

**Comment by nikolovskimilos** [comment]
@SutnuiBorshik You are right, this issue is resolved with new published version [2.1.0](https://www.npmjs.com/package/redistimeseries-js/v/2.1.0). 

The documentation has been misinterpreted, thanks for your help.

The following code:

```javascript

const RedisTimeSeries = require('../index');
const { Filter } = RedisTimeSeries;

const rtsClient = new RedisTimeSeries();

const start = async () => {
  await rtsClient.connect();

  const timestamp = Date.now();
  await rtsClient.add('key1', timestamp, 1).labels({ l1: 1 }).send();
  await rtsClient.add('key2', timestamp, 1).labels({ l1: 1, l2: 2 }).send();
  const data = await rtsClient.mrange(timestamp, timestamp + 1)
     .filter([
         Filter.equal('l1', 1),
         Filter.exists('l2'),
    ])
    .withLabels()
    .send();
  console.log(JSON.stringify(data));
};

start();
```

should return result like this

```
[["key2",[["l1","1"],["l2","2"]],[[1590656706892,"1"]]]]
```
